src/rt/aaA.d

```diff
@@ -684,6 +684,10 @@ extern (C) pure nothrow @nogc
                 break;
         }
     }
+    size_t _aaRangeLen(Range r)
+    {
+        return r.impl is null ? 0 : r.impl.length;
```

Comment:
@schveiguy Hmm, but Impl.length returns 'used - deleted', so I thought that's the one?